src/redux/watchList/actionCreator.js

```diff
@@ -0,0 +1,19 @@
+export const FETCH_WATCHLIST_SUCCESS = 'FETCH_WATCHLIST_SUCCESS';
+export const POST_WATCHLIST_SUCCESS = 'POST_WATCHLIST_SUCCESS';
+
+export function fetchWatchList() {
```

Comment:
be consequent, and use arrow function, our already exisitng actionCreators are arrow functions